index.html

```diff
@@ -454,6 +391,66 @@ <h4 id="git-linux">Linux</h4>
   </div>
 </div> <!-- End of 'Git' section. -->
 
+<div id="editor"> <!-- Start of 'editor' section. -->
+  <h3>Text Editor</h3>
+
+  <p>
+    When you're writing code, it's nice to have a text editor that is
+    optimized for writing code, with features like automatic
+    color-coding of key words.  The default text editor on Mac OS X and
+    Linux is usually set to Vim, which is not famous for being
+    intuitive.  if you accidentally find yourself stuck in it, try
+    typing the escape key, followed by ':q!' (colon, lower-case 'q',
```

Comment:
I'd keep the `<code>` wrapper around `:q!`, since it's something you're tying in your terminal.